packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -31,6 +32,29 @@ const Link = ({ to, onClick, ...props }) => {
     }
   `);
 
+  const properties =
+    typeof instrumentation === 'string'
+      ? { customProp: instrumentation }
+      : instrumentation;
+
+  const handleClick = useInstrumentedHandler(onClick, {
+    actionName: 'externalLink_click',
+    href: to,
+    ...properties,
```

Comment:
Since `Link` is used everywhere, most cases are not going to have an `instrumentation` prop defined. I _believe_ this would blow up when `instrumentation` is `undefined`.  Could you make sure this defaults to an empty object if its `undefined`?

Reply:
instrumentation is already defaulted to an empty object!! if you scroll up you can see that :)

Reply:
🤦‍♂️ Wow. I need to go outside and take a walk. Thanks!!